app/public/js/localSprites.js

```diff
@@ -147,7 +162,7 @@ function addLocalFallingSpikesSprite() {
         $.extend(rectangle(3 * 16, 14 * 16, 16, 16), {image: twilightTilesImage, hitBox}),
         $.extend(rectangle(2 * 16, 14 * 16, 16, 16), {image: twilightTilesImage, hitBox}),
     ];
-    var fallingSpikesSprite = new SimpleSprite({frames}, mainCharacter.x, cameraY - 32, 0, 5, 2, -2);
+    var fallingSpikesSprite = new SimpleSprite({frames}, mainCharacter.x, cameraY - 32, 0, 5, 2, -2); //what is this '5' in here?
```

Comment:
I just realized looking at this when I suggested commenting on PR that the term comment is ambiguous. In any case, when you go to open a PR, I suggest removing comments in the code like this and just adding them to the PR instead.

Reply:
Oh and to answer your question, that is the initial y velocity of the falling spike. Since nothing updates the falling spikes velocity, that causes it to just move steadily down the screen.